rpm/poppler.spec

```diff
@@ -101,7 +103,7 @@ and pdffonts (PDF font analyzer).
 %autosetup -p1 -n %{name}-%{version}/upstream
 
 %build
-%cmake . -G Ninja -DENABLE_BOOST=ON
+%cmake . -G Ninja -DENABLE_BOOST=ON -DENABLE_GPGME=off -DENABLE_QT6=off -DENABLE_LCMS=off -DCMAKE_CXX_FLAGS="-Wno-deprecated-declarations"
```

Comment:
For small detail, why the -Wno-deprecated-declarations? I'm not seeing much warnings without it either.

Reply:
There are only 10 or so... but somehow when i did the uplift the first time around they were fatal. Or maybe i was confused and managed to hide the true issue long enough to think this was a working fix.